src/lang/pt_PT.h

```diff
@@ -0,0 +1,205 @@
+#define D_OBJECT_UNKNOWN "Objeto desconhecido"
+#define D_OBJECT_MISMATCH "Os objetos não são iguais!"
+#define D_OBJECT_LOST "Objeto perdido!"
+#define D_OBJECT_CREATE_FAILED "Não foi possível criar o objeto %u"
+#define D_OBJECT_PAGE_UNKNOWN "A página ID %u não está definida"
+#define D_OBJECT_EVENT_UNKNOWN "Não se conhece o evento %d "
+
+#define D_ATTRIBUTE_UNKNOWN "Propiedade %s desconhecida"
+#define D_ATTRIBUTE_READ_ONLY "%s é de leitura apenas"
+#define D_ATTRIBUTE_PAGE_METHOD_INVALID "Não foi possível chamar %s numa página"
+
+#define D_OOBE_SSID_VALIDATED "SSID %s validado"
+#define D_OOBE_AUTO_CALIBRATE "Auto calibração ativada"
+#define D_OOBE_CALIBRATED "já se foi calibrado"
+
+#define D_DISPATCH_COMMAND_NOT_FOUND "Não se encontrou o comando '%s'"
+#define D_DISPATCH_INVALID_PAGE "Página inválida %s"
+#define D_DISPATCH_REBOOT "A reiniciar dispositivo!"
+
+#define D_JSON_FAILED "Não foi possível analisar o JSON:"
+#define D_JSONL_FAILED "A análise do JSONL falhou na linha %u"
+#define D_JSONL_SUCCEEDED "JSONL analisado"
+
+#define D_OTA_CHECK_UPDATE "A procurar a atualização do URL: %s"
+#define D_OTA_CHECK_COMPLETE "Verificação de atualização completa"
+#define D_OTA_CHECK_FAILED "Falhou a verificação de atualização: %s"
+#define D_OTA_UPDATE_FIRMWARE "Atualização de firmware OTA"
+#define D_OTA_UPDATE_COMPLETE "Atualização OTA completa"
+#define D_OTA_UPDATE_APPLY "A aplicar o novo firmware e reiniciar"
+#define D_OTA_UPDATE_FAILED "A atualização OTA falhou"
+#define D_OTA_UPDATING_FIRMWARE "A atualizar o firmware..."
+#define D_OTA_UPDATING_FILESYSTEM "A atualizar o sistema de ficheiros..."
+
+#define D_HTTP_HASP_DESIGN "Design do HASP"
+#define D_HTTP_INFORMATION "Informação"
+#define D_HTTP_HTTP_SETTINGS "Configuração HTTP"
+#define D_HTTP_WIFI_SETTINGS "Configuração Wifi"
+#define D_HTTP_MQTT_SETTINGS "Configuração MQTT"
+#define D_HTTP_GPIO_SETTINGS "Configuração GPIO"
+#define D_HTTP_MDNS_SETTINGS "Configuração mDNS"
+#define D_HTTP_TELNET_SETTINGS "Configuração Telnet"
+#define D_HTTP_DEBUG_SETTINGS "Configuração do debug"
+#define D_HTTP_GUI_SETTINGS "Configuração da GUI"
+#define D_HTTP_SAVE_SETTINGS "Guardar Configuração"
+#define D_HTTP_UPLOAD_FILE "Carregar ficheiro"
+#define D_HTTP_ERASE_DEVICE "Eliminar Configuração"
+#define D_HTTP_ADD_GPIO "Adicionar novo pino"
+#define D_HTTP_BACK "Atrás"
+#define D_HTTP_REFRESH "Atualizar"
+#define D_HTTP_PREV_PAGE "Página Anterior"
+#define D_HTTP_NEXT_PAGE "Página Seguinte"
+#define D_HTTP_CALIBRATE "Calibrar"
+#define D_HTTP_SCREENSHOT "Screenshot"
+#define D_HTTP_FILE_BROWSER "Editor de ficheiros"
+#define D_HTTP_FIRMWARE_UPGRADE "Atualização do firmware"
+#define D_HTTP_UPDATE_FIRMWARE "Atualizar o firmware"
+#define D_HTTP_FACTORY_RESET "Restauração de fábrica"
+#define D_HTTP_MAIN_MENU "Menu Principal"
+#define D_HTTP_REBOOT "Reiniciar"
+#define D_HTTP_CONFIGURATION "Configuração"
+#define D_HTTP_SENDING_PAGE "Foi enviado página %S a %s" // New

```

Comment:
```suggestion
#define D_HTTP_SENDING_PAGE "Foi enviado página %s a %s" // New

```

Reply:
There is actually one `%S` in the file. I need to verify if this is still correct, but `%S` should be fine in this line.